src/GlobalPayments.Api/Utils/Extensions.cs

```diff
@@ -206,5 +206,14 @@ public static class Extensions {
             }
             return rvalue;
         }
+        public static T[] SubArray<T>(this T[] data, int index, int length) {
+            T[] result = new T[length];
+            Array.Copy(data, index, result, 0, length);
+            return result;
+        }
+
+        public static bool IsNull<T>(this T data) {

```

Comment:
This doesn't take value types into consideration and what is the benefit of this statement over a standard null check?